Encrypted Ink/Services/MetamaskImporter.swift

```diff
@@ -0,0 +1,121 @@
+// Copyright © 2021 Encrypted Ink. All rights reserved.
+
+import SwiftStore
+import CryptoSwift
+import WalletCore
```

Comment:
Don't use WalletCore here.
Extract needed data and then handle private keys derivation right in WalletsManager.